### DISCUSSION TITLE
2019-10-01-orphaned-comments/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#27] [General]
# 2019-10-01-orphaned-comments/

Cjones85

https://fanzoflenazavaroni.github.io/2019-10-01-orphaned-comments/

**Comment by Cjones85** [comment]
Its been 4 years but this is me checkin in Mr. Rob. I hope you are well where ever you are.

**Reply by FanzOfLenaZavaroni** [reply]
Hi! Chris,
I'm fine thank you and as you can see this website is still going strong I hope things are going well for you.

As you have now seen I had to recreate visitors comments due to a system failure, thank you for taking the time to confirm your Orphaned Comments are real. It's a shame my side of the comments are lost as we had some honest chats about Lena's life.

**Reply by Cjones85** [reply]
Hello! I am happy that this website is still going. I keep it bookmarked and try to check in every once in a while, it could be a few years but I do check in! Lol! 

Yes, I noticed that you had a system failure. Thats VERY unfortuneate. We did have some good conversations on here about Lena. I took a look at all the orphaned comments and all of the ones attributed to me are mine. Thank you for trying to atleast make sure they were reposted!

**Reply by Cjones85** [reply]
Hello Mr. Rob. This is me checking in in 2026. I hope things are well with you and the site.

**Comment by FanzOfLenaZavaroni** [comment]
Hi Chris,

I'm fine and hope the same is true for you.

As for the site, it’s now in archive status. It’s been hard to keep the enthusiasm going because of the behaviour of some so‑called fans, which I’ve mentioned in various [notifications](https://fanzoflenazavaroni.github.io/2026-01-18-announcement/) around the site. I still add things now and then, but it’s challenging when all I have to work with is a date and Lena’s name.

Regards,
Robert

**Comment by Cjones85** [comment]
Yea, there some batshit crazy fans out there and a facebook group thats
gatekeeping Lenas story. I've had my runins and I have been warned to watch
what I say about Lena etc. Funny thing is, all I did was ask about how her
parents ever thought it was ok to let her move away from home with total
strangers and other things like that in an effort to bring to light Lenas
story as a cautionary tale to any parents thinking about thrusting their
young child into the lime light. That dorothy woman was pure evil. The more
I learned about Lena the more I am absolutly positive that her body issues
and annorexia came directly from dorothy putting her on diets, no doubt
convincing Lena she had to lose weight to be accepted. Anyways I could on
forever, lol. Suffice it to say, I no longer talk about Lena on facebook. I
do keep an eye on that facebook group from time to time but other than
Lenas birthday and death anniversaries they really don't post often anymore.
I'm glad this site is still here and you are still around holding it together
the best you can. I should check in more often really.

**Reply by FanzOfLenaZavaroni** [reply]
I agree; you only have to see my posts and the bio page, where I have commented on many things, to know where I stand. never understood the gate keeping as Lena was a public figure so why lock her behide closed groups and the like, my group was always public so if you joined great but if not could still see all posts anyway as of late can see why some switched to a closed group to prevent the explaotaion of content like I have on here am slowly removing all images so for some there is a good reason it more the fact your not free to give a personal view that is must questionable part.

Sadly, it's what fandom is now, not just Lena's, but any fandom, people seem to have forgotten that you can be nice, even friends and just agree to differ.
